src/org/pahappa/systems/registrationapp/services/UserService.java

```diff
@@ -1,5 +1,76 @@
 package org.pahappa.systems.registrationapp.services;
 
+import org.pahappa.systems.registrationapp.models.User;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
 public class UserService {
+    List<User> users = new ArrayList<User>();
+
+    public boolean registerUser(User user) {
+        try {
+            users.add(user);
+            return true;
```

Comment:
There is no validation to prevent users from registering with the same username. If 10 users are registered with the same username, what will uniquely identify the different users? Similarly, when retrieving users with a given username, which user will be retrieved since they all have the same username?